aide_design/cdc_functions.py

```diff
@@ -36,31 +34,48 @@ def _DiamTubeAvail(en_tube_series = True):
 #         return (1/16)*u.inch
 #==============================================================================
 
-NU_WATER = 1*(u.mm**2/u.s)
-
-@u.wraps(u.m**2/u.s, [u.kg/u.m**3], False)
-def _nu_alum(conc_alum):
-    nu = (1 + (4.255 * 10**-6) * conc_alum**2.289) * NU_WATER
-    return nu.to(u.mm**2/u.s)
-    
+@u.wraps(u.m**2/u.s, [u.kg/u.m**3, u.degK], False)
+def viscosity_kinematic_alum(conc_alum, temp):
+    """Return the dynamic viscosity of water at a given temperature.
+    
+    If given units, the function will automatically convert to Kelvin.
+    If not given units, the function will assume Kelvin.
+    This function assumes that the temperature dependence can be explained 
+    based on the effect on water and that there is no confounding effect from 
+    the coagulant.
+    """
+    nu = (1 + (4.255 * 10**-6) * conc_alum**2.289) * pc.viscosity_kinematic(temp).magnitude
     return nu
 
 
-@u.wraps(u.m**2/u.s, [u.kg/u.m**3], False)   
-def _nu_pacl(conc_pacl):
-    nu = (1 + (2.383 * 10**-5) * conc_pacl**1.893) * NU_WATER
-    return nu.to(u.mm**2/u.s)
+@u.wraps(u.m**2/u.s, [u.kg/u.m**3, u.degK], False)   
+def viscosity_kinematic_pacl(conc_pacl, temp):
+    """Return the dynamic viscosity of water at a given temperature.
     
+    If given units, the function will automatically convert to Kelvin.
+    If not given units, the function will assume Kelvin.
+    This function assumes that the temperature dependence can be explained 
+    based on the effect on water and that there is no confounding effect from 
+    the coagulant.
+    """
+    nu = (1 + (2.383 * 10**-5) * conc_pacl**1.893) * pc.viscosity_kinematic(temp).magnitude
     return nu
 
 
-@u.wraps(u.m**2/u.s, [u.kg/u.m**3, None], False)    
-def _nu_chem(conc_chem, en_chem):
-    if en_chem == 0:
-        return _nu_alum(conc_chem)
-        return _nu_pacl(conc_chem)
-    else:
-        return NU_WATER
+@u.wraps(u.m**2/u.s, [u.kg/u.m**3, u.degK, None], False)    
+def viscosity_kinematic_chem(conc_chem, temp, en_chem):
```

Comment:
Please justify the existence of this function. looks like it just hands off the parameters to the relevant function to handle based on an enumerator. This is OK, but a bit redundant. Please explain.